package.json

```diff
@@ -47,17 +48,18 @@
     "grunt-eslint": "^18.1.0",
     "grunt-npm": "^0.0.2",
     "grunt-simple-mocha": "^0.4.1",
-    "libxmljs": "^0.19.5",
     "load-grunt-tasks": "^3.2.0",
     "mocha": "^3.2.0",
     "nyc": "^15.0.0",
     "proxyquire": "^1.7.4",
     "sinon": "^1.17.3",
-    "sinon-chai": "^2.8.0"
+    "sinon-chai": "^2.8.0",
+    "fast-xml-parser": "^3.17.4"
```

Comment:
where are we using "fast-xml-parser"?

Reply:
A test, replaces `libxmljs` as that has not been updated in a year and they have not added support for newer Node versions;